news/2018-06-02-mappers-choice-awards-2017-results.md

```diff
@@ -0,0 +1,345 @@
+
+*written by [Tofu1222](https://osu.ppy.sh/u/6089608)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/garalulu.jpg)](https://osu.ppy.sh/u/757783)
+
+**[Garalulu](https://osu.ppy.sh/u/757783)** is making huge progress in both mapping and modding last year. With new 7K and 4K maps ranked, he showed us his overwhelming experience in creating maps of different key modes, manipulating SVs, and using different types of advanced patterns such as LN manipulation. Furthermore, his great modding skill helped him become a BN at the end of last year, which marks his knowledge in modding as well. That's why he can have the strength to take the Grand User Award for this mode.
+
+*written by [Tofu1222](https://osu.ppy.sh/u/6089608)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/kawawa.jpg)](https://osu.ppy.sh/u/4647754)
+
+As one of the oldest mapper and modder that is still active nowadays, **[Kawawa](https://osu.ppy.sh/u/4647754)** contributed many great 7K mapsets. Those sets with spreads provide the advancing players with increasing levels of practices, while the marathons, featured with delicate hitsounds and variated patterns, are loved by those seeking for challenges. It is of no surprise that his deep experience in mapping won him osu!mania mapper category.
+
+*written by [Tofu1222](https://osu.ppy.sh/u/6089608)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/maxus.jpg)](https://osu.ppy.sh/u/4335785)
+
+You can never find a more welcoming and active modder in the osu!mania mapping community than **[Maxus](https://osu.ppy.sh/u/4335785)**. His high activity in modding and BN work and his proficiency in mapping and guiding new mappers definitely paved the way for him to win the osu!mania modder category. 
+
+*written by [Tofu1222](https://osu.ppy.sh/u/6089608)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/protastic.jpg)](https://osu.ppy.sh/u/6712747)
+
+When we talked about the amount of high quality consistency in the mod contribution, **[Protastic101](https://osu.ppy.sh/u/6712747)** is undoubtedly one that we must mentioned. It's very occassional seeing someone in their BN era to be really passionate in keep helping new mapper on how they improve and patiently explain in a long length what the mapper must fix to achieve map in a higher quality. She is one of the pioneer that makes the ranked section much more accessible for the new mapper and certainly her contribution really deserves a few words than just an amazement that makes a huge sign in the mania history.
```

Comment:
must mentioned -> must mention
very occassional -> very common
-> passionate in helping new mappers improve and patiently explaining at length what the mapper could change to improve their map quality.
pioneer that makes -> pioneers that made
for the new mapper and certainly her contribution -> to new mappers. Her contribution
deserves a few words (etc.) -> deserves more than just a few words to describe the amazing contribution she made towards the gamemode
(there was some paraphrasing in the last one but it gets the same message across)